StorageTests/TestSQLiteHistory.swift

```diff
@@ -99,9 +106,8 @@ class BaseHistoricalBrowserTable {
 //
 // These tests snapshot the table creation code at each of these points.
 
-class BrowserTableV6: BaseHistoricalBrowserTable {
-    var name: String { return "BROWSER" }
-    var version: Int { return 6 }
+class BrowserSchemaV6: BaseHistoricalBrowserSchema {
```

Comment:
This is a tricky one. It's really hard for you to continue testing the code that we're trying to test if you make changes like this — you're essentially creating a database layout that never existed (v6 but not using `SchemaTable`)!

At this point you might be best off running these tests, capturing the intermediate databases, and checking them in as fixtures.